README.adoc

```diff
@@ -5,3 +5,113 @@ This is a Kubernetes Operator to manage Apache Kafka clusters.
 It is written by https://www.stackable.de[Stackable] in Rust, and it is supposed to be used with the https://github.com/stackabletech/agent[Stackable Agent] instead of the Kubernetes kubelet.
 
 The docs can be found in the `docs` subdirectory, and they are published together with docs for all other Stackable products at https://docs.stackable.tech.
+
+This operator can work with the https://github.com/stackabletech/zookeeper-operator[Stackable Operator for Apache ZooKeeper] for ZooKeeper support and https://github.com/stackabletech/opa-operator[Stackable Operator for OPA] for OpenPolicyAgent support.
+
+== Building
+
+This operator is written in Rust.
+It is developed against the latest Rust release (1.53 at the time of writing this).
+
+    cargo build
+
+== Configuration Options
+
+The cluster can be configured via a YAML file. This custom resource specifies the amount of replicas for each role group, role group or role specific configuration like port definitions etc.
+
+    apiVersion: kafka.stackable.tech/v1alpha1
+    kind: KafkaCluster
+    metadata:
+      name: simple
+    spec:
+      version:
+        kafka_version: 2.8.0
+      zookeeperReference:
+        namespace: default
+        name: simple
+      opa:
+        reference:
+          namespace: default
+          name: simple-opacluster
+        authorizerClassName: com.bisnode.kafka.authorization.OpaAuthorizer
+        authorizerCacheInitialCapacity: 0
+        authorizerCacheMaximumSize: 0
+        authorizerCacheExpireAfterSeconds: 0
+      brokers:
+        roleGroups:
+          default:
+            selector:
+              matchLabels:
+                kubernetes.io/arch: stackable-linux
+            replicas: 1
+            config:
+              logDirs: "/tmp/kafka-logs"
+              metricsPort: 9606
```

Comment:
These sections are also in the `docs` folder. I would remove them from the readme file so that we do not have to maintain duplicated documentation.

Furthermore I would move the following sections also to the `docs`.